src/main/java/org/spongepowered/common/mixin/core/util/ClassInheritanceMultiMapMixin.java

```diff
@@ -34,9 +34,13 @@
 
     /**
      * @author Minecrell
+     * @author i509VCB - February 23rd, 2020 - 1.14.4
+     *
+     * @todo Is this even necessary anymore?
```

Comment:
Yes, this is still necessary. Since the logic is currently looking at the passed in class, we want to be able to ask if the class is either an interface OR an extended class of the base class.

Reply:
Will do once I pack since I have to go to Dallas tomorrow